pyproject.toml

```diff
@@ -86,6 +86,7 @@ ignore = [
     "SIM117",
     "SIM118",
     "UP007", # keep-runtime-typing
+    "C416", # tuple(generator) is a performance regression
```

Comment:
This isn't C416 at all, it's preview behavior of C409 https://docs.astral.sh/ruff/rules/unnecessary-literal-within-tuple-call/